### PR TITLE
fix(DDI-960): Modified the container storybook page

### PR DESCRIPTION
**Pull request by ArakTaiRoth**
Made sure that the basic container had no accent bar. Changed default container to basic container.
Added title to the properties tabs
Modified headingSize to accentBar.
Modified colored to backgroundColour.
Also updated the figma link in the one-column and two-column layout pages.

### CODE REVIEW

File: apps/angular-demo/src/app/container/container.component.html [hunk 1]
```diff
@@ -50,15 +50,14 @@ <h2>Nested</h2>
     <goa-button type="primary" size="compact">Assign to me</goa-button>
   </div>
   <h2>Custom H2</h2>
-  <goa-container type="primary" headingsize="small">
+  <goa-container accent="thing">
```
Comment (chrisolsen) [inline apps/angular-demo/src/app/container/container.component.html:53]: misspelling

File: libs/docs/src/components/common/Container.stories.mdx [hunk 2]
```diff
@@ -37,127 +37,87 @@ import {
...
     defaultValue="relaxed"
-    description="sets the amount of white space in the container"
+    description="Sets the amount of white space in the container"
+  />
+  <Prop
+    name="title"
+    type="slot"
+    lang="angular"
+    defaultValue=""
+    description="Sets the content in the top left of the accent bar"
```
Comment (chrisolsen) [inline libs/docs/src/components/common/Container.stories.mdx:60]: Do we want to say `top left` or `left` in the accent bar?

File: libs/docs/src/components/common/Container.stories.mdx [hunk 3]
```diff
@@ -166,29 +126,89 @@ Non-interactive containers indicate that the content is only for the user to rev
...
+          <h2>Success Container w/ Header</h2>
+          If you use an accent, the background colour won't be filled in
+        </GoAContainer>
+      `}
     />
   </Tab>
 </Tabs>
 
-#### Heading bar types (large, small, with text)
+#### Heading bar types (thick, thin, with text)
```
Comment (chrisolsen) [inline libs/docs/src/components/common/Container.stories.mdx:209]: Is this now the accent bar?

File: apps/angular-demo/src/app/container/container.component.html (hunk 1 above)
Comment (ArakTaiRoth) [reply apps/angular-demo/src/app/container/container.component.html]: Changed to `thin`

File: libs/docs/src/components/common/Container.stories.mdx (hunk 2 above)
Comment (ArakTaiRoth) [reply libs/docs/src/components/common/Container.stories.mdx]: Changed to `left`, and `right` on the other slot

File: libs/docs/src/components/common/Container.stories.mdx (hunk 3 above)
Comment (ArakTaiRoth) [reply libs/docs/src/components/common/Container.stories.mdx]: Modified to say `Accent bar types`

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**